src/concepts/essentials/workflows/nodes/coding_nodes/convert_coding_nodes.md

```diff
@@ -0,0 +1,116 @@
+# Convert Workflow Coding Nodes
```

Comment:
Unsure but I feel this is more a reference than a concept (it's in between really).
I think I would put it in reference, cut some things like how the console works (that is more concept) and create a longer concept page on the various element (like the SDK alone should be a page).